backend/src/main/java/se/chalmers/dat265/group1/model/PhysicalPart.java

```diff
@@ -0,0 +1,35 @@
+package se.chalmers.dat265.group1.model;
+
+import java.util.LinkedList;
+import java.util.List;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.List'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)